book/routing.rst

```diff
@@ -1429,7 +1441,10 @@ Generating URLs with Query Strings
 The ``generate`` method takes an array of wildcard values to generate the URI.
 But if you pass extra ones, they will be added to the URI as a query string::
 
-    $this->get('router')->generate('blog', array('page' => 2, 'category' => 'Symfony'));
+    $this->get('router')->generate('blog', array(
+        'page' => 2,
+        'category' => 'Symfony'
```

Comment:
A quick question: do we add a trailing comma for the last array element in the docs ... or do we only add it in code?